core/domain/acl_decorators.py

```diff
@@ -224,6 +351,17 @@ def can_upload_exploration(handler):
     """Decorator that checks if the current user can upload exploration."""
 
     def test_can_upload(self, **kwargs):
+        """Check of the user can upload exploration.
```

Comment:
Make it consistent.
Somewhere it's `Check of ...`, `Check if ....` & `Check whether .....`

Reply:
Done :+1: